### ISSUE TITLE
[BUG] Can't use `require('pixi-filters.js')`

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi-filters#159]
in the dist file `pixi-filters.js` , there is one line : 
```
Object.assign(PIXI.filters,this.__filters);
```
when use  `require('pixi-filters.js')` , the `this` is undefined, then there is an error.

**Comment by finscn** [comment]
the code in pixi-filters.js  is like this : 
```
var __filters = {
 // sth.
};
Object.assign(PIXI.filters,this.__filters);
```

the `this` is wrong. in Strict Mode , the `this` is `undefined`.
it should be 
```
var __filters = {
 // sth.
};
Object.assign(PIXI.filters,__filters);  // no `this`
```

**Comment by bigtimebuddy** [comment]
Fixed with #160
